### ISSUE TITLE
Alarm tests fail on EC2

### ISSUE THREAD

**Issue by emiltin** [rsmp-nordic/rsmp_validator#78]
Tests that depend on triggered an alarm fails on the EC2. Can alarm be trigged on the EC2?

**Comment by otterdahl** [comment]
Yes they can. By setting the lamp reference values really low using the web interface. This should create lamp errors.

However, in the test ec2 I can't see any lamps.
